content/id/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Memperluas Kubernetes API dengan Lapisan Agregasi
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Lapisan agregasi memungkinkan kubernetes untuk diperluas dengan API tambahan, di luar dari apa yang ditawarkan oleh inti API kubernetes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Ikhtisar
+Lapisan agregasi memungkinkan instalasi tambahan ragam API _Kubernetes-style_ di cluster kamu. Ini bisa berupa bawaan, maupun solusi dari pihak ke-3 yang sudah ada, seperti [layanan-katalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/README.md), atau API yang telah dibuat pengguna seperti [pembangun-apiserver](https://github.com/kubernetes-incubator/apiserver-builder/blob/master/README.md), yang dapat membantu kamu memulai.
```

Comment:
```suggestion
Lapisan agregasi memungkinkan instalasi tambahan beragam API _Kubernetes-style_ di kluster kamu. Tambahan-tambahan ini dapat berupa solusi-solusi yang sudah dibangun (_prebuilt_) oleh pihak ke-3 yang sudah ada, seperti [service-catalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/README.md), atau API-API yang dibuat oleh pengguna seperti [apiserver-builder](https://github.com/kubernetes-incubator/apiserver-builder/blob/master/README.md), yang dapat membantu kamu memulainya.
```

- Keep plural forms.
- `. . .pre-built, existing 3rd party solutions,. . .` refers to the same thing, not two different things.
- `These can be pre-built. . .` -> plural. To emphasize what it refers to (the thing from previous word which is additional APIs), we can use `tambahan-tambahan ini`.
- `user-created` is a passive form, not a passive perfect form. We can omit the "telah"
- `service-catalog` -> Actually this is the name of an extension API. From the doc: `Service Catalog installs the servicecatalog.k8s.io API and provides . . .`. Let's keep it as is. https://kubernetes.io/docs/concepts/extend-kubernetes/service-catalog/
- `apiserver-builder` -> This is also a name of a collection of libraries to build apiserver. Let's keep it as is to avoid misinterpretation/confusion.

Reply:
Addressed here https://github.com/kubernetes/website/pull/16903/commits/6474e5e14057355e42e529f07f419ffed2a5a3bd